nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -37,6 +37,12 @@ let
 in
 
 {
+  imports = [
+    (mkRemovedOptionModule [ "services" "xserver" "displayManager" "gdm" "autoLogin" "enable" ]
+      "Set the option `services.xserver.displayManager.autoLogin.enable' instead.")
+    (mkRemovedOptionModule [ "services" "xserver" "displayManager" "gdm" "autoLogin" "user" ]
+      "Set the option `services.xserver.displayManager.autoLogin.user' instead.")
+  ];
```

Comment:
If we would include in to 20.03 (it is not too late, right?), we do not need these `mkRemovedOptionModule` because these options have been introduced few weeks ago and did not yet land to any stable release (20.03 is going to be the first)

Reply:
20.03 is a GO https://discourse.nixos.org/t/go-no-go-meeting-nixos-20-03-markhor/6495/23?u=worldofpeace. So this change won't see that release.